.github/workflows/llm-binary-build.yml

```diff
@@ -312,6 +319,11 @@ jobs:
     needs: check-windows-avx2-vnni-artifact
     if: needs.check-windows-avx2-vnni-artifact.outputs.if-exists == 'false'
     steps:
+      - uses: actions/checkout@v3
```

Comment:
We don't need a checkout for this job. The following code contains the procedure for checking out llm.cpp.